packages/shared/components/MenuLogin/MenuLogin.story.tsx

```diff
@@ -20,8 +20,24 @@ import { Flex } from 'design';
 import { MenuLogin, MenuLoginAsync } from './MenuLogin';
 import { LoginItem } from './types';
 import { makeProcessingAttempt } from 'shared/hooks/useAsync';
+import { MenuLoginTheme } from 'teleterm/ui/DocumentCluster/ClusterResources/MenuLoginTheme';
 
 storiesOf('Shared/MenuLogin', module).add('MenuLogin', () => {
+  return <MenuLoginExamples />;
+});
+
+storiesOf('Shared/MenuLogin', module).add(
+  'MenuLogin in Teleport Connect',
+  () => {
+    return (
+      <MenuLoginTheme>
+        <MenuLoginExamples />
+      </MenuLoginTheme>
+    );
+  }
+);
```

Comment:
Added those to let other teammates modifying `MenuLogin` know that Teleport Connect uses a special variant of it.

Reply:
It's a special case just for the storybook, so I feel that's fine. 🤔 The storybook files are completely different entrypoints than the rest of the code.

I could move it under the Teleterm section, but then it would be easy to miss and thus didn't accomplish its original purpose. Would you agree that this makes the "MenuLogin in Teleport Connect" example justified here?

Reply:
A part of me still feels a little wrong 😄, but since it's a non-production code, let's do it. I can agree that moving it under Teleterm section doesn't make much sense.